js/index.js

```diff
@@ -0,0 +1,63 @@
+$(document).ready(function() {
+
+    // adds operand or operator to screen
+
+    $('.buttons > span').click(function() {
+        if (isDone) {
+            reset()
+        }
+        $('#screen').append($(this).text())
+    });
+
+    $('#zero').click(function() {
+        if (isDone) {
+            reset()
+        }
+        $('#screen').append($(this).text())
```

Comment:
there are a few "appends" which will work in this case. Just as a general note, we want to use append when we're actually adding HTML elements, not just text. you can use .text() to set text TO #screen the same way you can use it to pull it out of $(this)